easybuild/easyconfigs/q/Qt5/Qt5-5.14.1_fix-mantissatable.patch

```diff
@@ -0,0 +1,21 @@
+# replaces F16C preprocessor conditional directives in gen_qfloat16_tables.cpp 
+# author: Jiri Furst, fix taken from Zdenek Matej's pr #10425
```

Comment:
@furstj Is there a public URL for the PR you're referring to? If so, please include it (and update the SHA256 checksum for the patch accordingly).

Reply:
Ah, you meant PR #10425 in this repo...

In that case, it's better to include https://github.com/easybuilders/easybuild-easyconfigs/issues/10332 as reference in the patch description imho...

Reply:
The reference has been changed in commit https://github.com/easybuilders/easybuild-easyconfigs/pull/11063/commits/eb1fff89040d8137cdbd8e8725eaf30b971f6fa0, see bellow.